go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/AlecAivazis/survey/v2 v2.2.15
 	github.com/PuerkitoBio/goquery v1.8.1
 	github.com/aws/aws-sdk-go v1.45.26
+	github.com/aws/aws-sdk-go-v2 v1.24.1
```

Comment:
@hunterkepley Is this intentional?

Reply:
nope :thinking: not sure why thats there